packages/react/src/analytics/integrations/Riskified/Riskified.ts

```diff
@@ -0,0 +1,113 @@
+import {
+  EventData,
+  integrations,
+  LoadIntegrationEventData,
+  StrippedDownAnalytics,
+  TrackTypes,
+  TrackTypesValues,
+  utils,
+} from '@farfetch/blackout-analytics';
+import get from 'lodash/get';
+import type { RiskifiedIntegrationOptions } from './types';
+
+class Riskified extends integrations.Integration<RiskifiedIntegrationOptions> {
+  baseUrl!: string;
+  isReadyPromise: Promise<unknown>;
+  resolveIsReadyPromise?: (value?: unknown) => void;
+
+  /**
+   * Creates an instance of Riskified and sets the script on the page.
+   *
+   * @param options - Custom options for the integration.
+   * @param loadData - Analytics's load event data.
+   * @param analytics - Stripped down analytics instance.
+   */
+  constructor(
+    options: RiskifiedIntegrationOptions,
+    loadData: LoadIntegrationEventData,
+    analytics: StrippedDownAnalytics,
+  ) {
+    super(options, loadData, analytics);
+
+    this.resolveIsReadyPromise = undefined;
+    this.isReadyPromise = new Promise(resolve => {
+      this.resolveIsReadyPromise = resolve;
+    });
+
+    if (!options.shop) {
+      utils.logger.warn(
```

Comment:
I am not sure if I have talked about this before or not (my apologies if I have not) but we should throw in this case since without the `shop` property this instance will not have set up itself correctly, so it should not take space in the `activeIntegrations` array in analytics.